gpkit/tests/t_geometric_program.py

```diff
@@ -31,10 +32,9 @@ def test_trivial_gp(self):
         """
```

Comment:
Renamed `t_geometric_program`, but kept TestGP, since it makes sense to distinguish the tests between GPs and SPs